test/functional/styles/icons.js

```diff
@@ -0,0 +1,90 @@
+const variants = [
```

Comment:
let's `require` the `ecl-icons.config.js` directly and filter this so to avoid maintaining 2 definition sets?